packages/datawell/src/css/index.js

```diff
@@ -5,9 +5,7 @@ export default {
   '.psds-datawell': {
     borderLeft: `1px solid ${core.colors.gray03}`,
     minHeight: '104px',
-    padding: `0 ${core.layout.spacingMedium} ${core.layout.spacingMedium} ${
-      core.layout.spacingLarge
-    }`
+    padding: `0 ${core.layout.spacingMedium} ${core.layout.spacingMedium} ${core.layout.spacingLarge}`
```

Comment:
I think that even small things like formatting should be kept out of a PR if it's for an unrelated thing.  It becomes noise.  And we shouldn't need to test the other thing with our eyes in a code review or by running code to make sure in the context of a PR.  Nbd, but I think it would be helpful.

Reply:
yeah i agree. i'm unsure why this got changed...maybe my new editor setup isn't working as expected - will investigate